selfdrive/ui/qt/offroad/onboarding.cc

```diff
@@ -67,31 +72,26 @@ TrainingGuide::TrainingGuide(QWidget* parent) {
 
 
 QWidget* OnboardingWindow::terms_screen() {
-
-  QGridLayout *main_layout = new QGridLayout();
-  main_layout->setMargin(100);
-  main_layout->setSpacing(30);
-
-  main_layout->addWidget(title_label("Review Terms"), 0, 0, 1, -1);
-
-  QLabel *terms = new QLabel("See terms at https://my.comma.ai/terms");
-  terms->setAlignment(Qt::AlignCenter);
-  terms->setStyleSheet(R"(
-    font-size: 75px;
-    border-radius: 10px;
-    background-color: #292929;
-  )");
-  main_layout->addWidget(terms, 1, 0, 1, -1);
-  main_layout->setRowStretch(1, 1);
-
+  QVBoxLayout *main_layout = new QVBoxLayout;
+  main_layout->setContentsMargins(40, 0, 40, 0);
+  QWebEngineView *view = new QWebEngineView(this);
```

Comment:
Think this slows ui start a lot. We should only load chromium when the terms actually have to be displayed.

Reply:
Just make sure the widget is never created if we're not going to show it.

Reply:
Fixed